Libraries/Components/ScrollView/ScrollView.windows.js

```diff
@@ -410,11 +413,11 @@ const ScrollView = React.createClass({
   },
 
   mixins: [ScrollResponder.Mixin],
-
   _scrollAnimatedValue: (new Animated.Value(0): Animated.Value),
   _scrollAnimatedValueAttachment: (null: ?{detach: () => void}),
   _stickyHeaderRefs: (new Map(): Map<number, ScrollViewStickyHeader>),
   _headerLayoutYs: (new Map(): Map<string, number>),
+
```

Comment:
@matthargett this PR consists of cherry-picking some commits from RNW master. It looks like the style changes came from the `createClass` migration commit (ae0f9cd42c4034f4ee9f683cddc054e6ae96121a). I'm not sure if they came from taking code from FB's repo.

Because this is just a cherry-pick of what's in master, I'm in favor of leaving the commit as-is unless you have specific concerns. If I have to remove the style changes from this PR, I'm afraid it'll take time to do the change, test it, and there's a risk I'll introduce regressions while modifying the commit.